manifests/ha/base/redis-ha/generate.sh

```diff
@@ -14,3 +14,4 @@ helm template argocd ./chart \
 sed -e 's/check inter 1s/check inter 3s/' ./chart/upstream_orig.yaml >> ./chart/upstream.yaml && rm ./chart/upstream_orig.yaml
 sed -i.bak 's/timeout server 30s/timeout server 6m/' ./chart/upstream.yaml && rm ./chart/upstream.yaml.bak
 sed -i.bak 's/timeout client 30s/timeout client 6m/' ./chart/upstream.yaml && rm ./chart/upstream.yaml.bak
+sed -i.bak -E 's/^([[:space:]]){8}sentinel replaceme argocd/    bind/' ./chart/upstream.yaml && rm ./chart/upstream.yaml.bak
```

Comment:
I am not proud of this but I am not sure of a better way. The templating doesn't support just [adding "raw" config values](https://github.com/DandyDeveloper/charts/blob/5a93b9545a4791321cf3762ff85338afa119672c/charts/redis-ha/templates/_configs.tpl#L59) like Redis.

Reply:
It works :)